statistics/urls.py

```diff
@@ -11,6 +11,9 @@
     url(r'^tutorial-content/$', 'statistics.views.tutorial_content', name="statistics_content"),
     url(r'^training/(?P<rid>\d+)/participants/$', 'statistics.views.training_participant',
         name="statistics_training_participants"),
+    url(r'^training/(?P<rid>\d+)/participant/$', 'statistics.views.studentmaster_ongoing',
+        name="statistics_studentmaster_ongoing"),
```

Comment:
`r'^training/(?P<rid>\d+)/participant/$'` is not very schematic, with what it's trying to display and quite confusing as there is already a `training/(?P<rid>\d+)/participants/` url.

Probably you want to have something like.


`/statistics/training/23/students/ongoing`